docs/changelog.md

```diff
@@ -171,6 +171,24 @@ A new optimization has been added to compute aggregations (min, max, sum and avg
 If the index backend is Elasticsearch, a `double` value is used to hold the result. As a result, aggregations on long numbers greater than 2^53 are approximate.
 In this case, if the accurate result is essential, the optimization can be disabled by removing the strategy `JanusGraphMixedIndexAggStrategy`: `g.traversal().withoutStrategies(JanusGraphMixedIndexAggStrategy.class)`.
 
+##### Breaking change for transaction logs processing
+
+[Transaction Log](advanced-topics/transaction-log.md) processing has a breaking change.  
```

Comment:
This only affects transaction logs started by users, right? I think so but just wanted to make sure.